HW2/P3/mandelbrot.pyx

```diff
@@ -31,16 +70,74 @@ cpdef mandelbrot(np.complex64_t [:, :] in_coords,
     assert in_coords.shape[0] == out_counts.shape[0], "Input and output arrays must be the same size"
     assert in_coords.shape[1] == out_counts.shape[1],  "Input and output arrays must be the same size"
 
+    # convert to complex part and real part
+    in_reals = np.real(in_coords)
+    in_imags = np.imag(in_coords)
+    
+    num_threads = 4
+     
+    # for reseting purposes
+    zeros8 = AVX.float_to_float8(0.0)
+
+    # used in comparison later
+    fours8 = AVX.float_to_float8(4.0)
+    nans8 = AVX.greater_than(fours8,zeros8)
```

Comment:
Because (using 32 bit floats) 1.0 (decimal) = 00000000000000000000000000000001(binary)

The value I computed is 11111111111111111111111111111111(binary)